packages/integrations/cloudflare/src/server.advanced.ts

```diff
@@ -44,19 +44,6 @@ export function createExports(manifest: SSRManifest) {
 				request.headers.get('cf-connecting-ip')
 			);
 
-			// `getRuntime()` is deprecated, currently available additionally to new Astro.locals.runtime
-			// TODO: remove `getRuntime()` in Astro 3.0
-			Reflect.set(request, Symbol.for('runtime'), {
-				env,
-				name: 'cloudflare',
-				caches,
-				cf: request.cf,
-				...context,
-				waitUntil: (promise: Promise<any>) => {
-					context.waitUntil(promise);
-				},
-			});
-
```

Comment:
Is this safe at this point? I expect users to be already depending on it in their astro 3.0 projects.

Reply:
As `runtime.ts` files was removed last week, `getRuntime()` wouldn't work at all. :)

Reply:
OK thanks for clarifying